plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -779,9 +779,11 @@ codewhisperer.explorer.enabled=\   Enabled
 codewhisperer.explorer.inline_suggestions=Inline Suggestions
 codewhisperer.explorer.learn=Learn
 codewhisperer.explorer.pause_auto=Pause Auto-Suggestions
+codewhisperer.explorer.pause_auto_scans = Pause Auto-Scans
```

Comment:
scan or scans

Reply:
Scans. Like Tai used for VSCode: https://github.com/aws/aws-toolkit-jetbrains/pull/4211#discussion_r1543505646